lib/configParser.ts

```diff
@@ -0,0 +1,258 @@
+import { resolve, dirname } from 'path';
```

Comment:
consider running all your .ts files through clang-format so I don't need to nitpick about whitespace :)

You can see in other repos (eg. angular/ts2dart) how we've hooked up clang-format so it's convenient to run and enforced by the build.